wv-wild-web/docs/GBP_CHECKLIST.md

```
@@ -0,0 +1,192 @@
+# Google Business Profile Optimization Checklist
+
+**For Kim at WV Wild Outdoors**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)